website/src/roadmap/go-sdk.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: section
+title: "Go SDK Roadmap"
+permalink: /roadmap/go-sdk/
+section_menu: section-menu/roadmap.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Go SDK Roadmap
```

Comment:
@herohde @lostluck I didn't write much but just copied the bits from "ongoing projects". If there is something of interest to users here, please recommend

Reply:
http://apache-beam-website-pull-requests.storage.googleapis.com/6718/roadmap/go-sdk/index.html

Reply:
Same, LGTM for now.